docs/configuration_and_usage.rst

```diff
@@ -99,6 +99,77 @@ to the help strings provided by the form.
 You can preview emails in the template edit view using the "Preview (program)" and "Preview (course)" buttons in
 top-right corner.
 
+Integrated Channels
+-------------------
+
+SAP SuccessFactors Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One of the integrated channels available for an ``EnterpriseCustomer`` is `SAP SuccessFactors`_.
+
+To integrate an ``EnterpriseCustomer`` with SAP SuccessFactors:
+
+* Set up a `TPA SAML Integration`_ for the SAP SuccessFactors endpoint.
+* From the LMS Admin, ensure that your ``EnterpriseCustomer`` record has its ``identity_provider`` field pointing to the
+  SAML Integration created in the previous step.
+
+  This allows learners to create accounts using Single Sign-On from SAP SuccessFactors, and to have their course
+  completion records sent back to SAP SuccessFactors.
+* From the LMS Admin, create an ``SAP SuccessFactors Enterprise Customer Configuration`` record, and link it to your
+  ``EnterpriseCustomer``.
+
+  Ensure that the record is marked ``Active`` to allow the management commands below to send data via this channel.
+
+.. _SAP SuccessFactors: https://www.successfactors.com
+.. _TPA SAML Integration: http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/tpa/tpa_SAML_SP.html
+
+Management Commands
+^^^^^^^^^^^^^^^^^^^
+
+These Django management commands send data to ``EnterpriseCustomer``'s active, integrated channels.
+
+They must be run from the command line on a server with the Open edX environment installed.
+
+.. note::
+
+   If data sharing consent is enabled for your ``EnterpriseCustomer``, then any Enterprise learners enrolled in the
+   associated courses must consent to data sharing, thus permitting their data to be sent back to SAP SuccessFactors.
+
+   If data sharing consent is *not* enabled for your ``EnterpriseCustomer``, then learner data may be sent to SAP
```

Comment:
This sounds like a good idea to me, however today there were some discussions regarding exactly when we need to gather consent and when it can be assumed that consent has been granted due to the employee-employer relationship.  I believe we ended up agreeing that consent should be gathered on the edX side regardless of the status of E2E (@griffresch please confirm here or on ENT-259).  

So, I *think* I'm okay with limiting this process to transmit only those learners who have explicitly granted consent, although I think the overall on/off flag for this feature should also be one of the gates.  That is, if consent is not enabled, we should not be sending any data at all.  If we allow data to be shared with consent disabled, that seems like an end-around to the whole process.  I'll defer to @griffresch for the final say on this, however.